plugin.program.autocompletion/plugin.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf8 -*-
+
+# Copyright (C) 2015 - Philipp Temminghoff <phil65@kodi.tv>
+# This program is Free Software see LICENSE file for details
+
+import sys
+import xbmc
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+import AutoCompletion
+
+ADDON = xbmcaddon.Addon()
+ADDON_VERSION = ADDON.getAddonInfo('version')
+
+
+def get_kodi_json(method, params):
+    json_query = xbmc.executeJSONRPC('{"jsonrpc": "2.0", "method": "%s", "params": %s, "id": 1}' % (method, params))
+    return json.loads(json_query)
```

Comment:
Is it me, or are we missing an `import json` here?

Reply:
True, I guess I haven't noticed it because it gets imported in the AutoCompletion module already. Anyway. Added it.